### DISCUSSION TITLE
MiniO keep writing something to disk so that make my HDD keep spinning

### DISCUSSION THREAD

**Discussion by YYYSSG** [minio/minio#17928] [Q&A] [answered]
I've noticed that a 4KB file called `/data/.minio.sys/tmp/d2a70d25-9b92-4955-93e7-e13cf654af11` keeps getting modified for as long as a minute.
The problem is that I have mounted `/data` with that file on the HDD, so modifying that file is causing the HDD to keep making noise.
Is there any way to fix this? The network was not being used when that file was written.

**Comment by harshavardhana** [comment, marked as answer]
We actively monitor the disk, if you want this to be offline please use 

export _MINIO_DISK_ACTIVE_MONITORING=off

**Reply by YYYSSG** [reply]
Thank you for your answer.
The 4KB file is gone, but the container is still using the disk every minute or so, is there anything else I can do to prevent this?

**Reply by klauspost** [reply]
No. The scanner will run continuously.
